src/Workspaces/CSharp/Portable/Extensions/ArrowExpressionClauseSyntaxExtensions.cs

```diff
@@ -48,7 +48,7 @@ internal static class ArrowExpressionClauseSyntaxExtensions
             return true;
         }
 
-        private static StatementSyntax ConvertToStatement(
+        public static StatementSyntax ConvertToStatement(
```

Comment:
This is obviously not ideal... The thing is that this method is also useful for converting expression lambdas to statements, not just expression-bodied members. Maybe it could be moved to ExpressionSyntaxExtensions and then there would be a separate class for lambda extensions? I didn't want to restructure the code significantly in my first contribution, especially in layers below that I wasn't supposed to touch, just to make a simple change in a refactoring.

Reply:
i would leave it here for now.